### ISSUE TITLE
kiosk menu is getting too long

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2106]
I knew it would happen one day and we are close: The combination of the main and sub-menu in one menu is making the menu itself too long. I think it would be best to move the sub-menu into its own control. Perhaps a sidebar or so?

**Comment by luizaogs** [comment]
I was going to open a ticket for this but found this old one. The hub menu for admins is becoming a bit dizzying :)

**Comment by urapadmin** [comment]
true, but at least I already did something so that it fits better. Which might need a cache refresh.
The number of options is a bit dizzying at this point.
In the admin menu we could simply get rid of all the sub-menu items that simply lead to the UI with all the lists (the one we have no name for). That would reduce it. 

- I'll definitely get rid of the kiosk query and qc rules menu items - nobody is using them.
